ci/fsc-fts-update.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I believe this script belongs to the fts repo

Reply:
I really meant to have a script that updates FTS with the current FSC commit.

Reply:
I understand. However, I think this script should move to the FTS repo as it creates a PR on the FTS repo.

Such a script could support multiple strategies:
- Get the commit hash from any local git clone / working branch.
- Get the lastest commit hash from main
- Provide the commit hash as a parameter to the script and update all go.mod files inside the project.

I believe the right way would be to use dependabot and schedule a daily check if the main branch on FSC has something new. As dependabot only "detects" versioned updated (not individual commits) it won't work like that. You can create a GH action that is fired once per day to check if there is a new commit on FSC main branch and then create a PR.

Reply:
okay, I'll remove the script.